packages/@aws-cdk/aws-apigateway/test/test.method.ts

```diff
@@ -693,6 +693,57 @@ export = {
       });
     }, /Authorization type is set to NONE which is different from what is required by the authorizer/);
 
+    test.done();
+  },
+  'method has Auth Scopes'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const api = new apigw.RestApi(stack, 'test-api', { cloudWatchRole: false, deploy: false });
+
+    // WHEN
+    new apigw.Method(stack, 'my-method', {
+      httpMethod: 'POST',
+      resource: api.root,
+      options: {
+        apiKeyRequired: true,
+        authorizationScopes: ['AuthScope1', 'AuthScope2'],
+      }
+    });
+
+    // THEN
+    expect(stack).to(haveResource('AWS::ApiGateway::Method', {
+      ApiKeyRequired: true,
+      AuthorizationScopes: ['AuthScope1', 'AuthScope2']
+    }));
+
+    test.done();
+  },
+  'use default Auth Scopes'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const api = new apigw.RestApi(stack, 'test-api', {
+      cloudWatchRole: false,
+      deploy: false,
+      defaultMethodOptions: {
+        authorizationScopes: ['DefaultAuth']
+      }
+    });
+
+    // WHEN
+    new apigw.Method(stack, 'defaultAuthScopes', {
+      httpMethod: 'POST',
+      resource: api.root,
+      options: {
+        operationName: 'defaultAuthScopes'
+      }
+    });
+
+    // THEN
+    expect(stack).to(haveResource('AWS::ApiGateway::Method', {
+      OperationName: 'defaultAuthScopes',
+      AuthorizationScopes: ['DefaultAuth']
+    }));
+
```

Comment:
2 test cases missing -

1. given no authorization scopes, verify that the property is absent. Use [`ABSENT` feature](https://github.com/aws/aws-cdk/tree/master/packages/%40aws-cdk/assert#check-existence-of-a-resource) of the assert library.

2. given authorization scope in method options and default method options, the one in method options is picked up.

Reply:
Updated as per comments.